src/MOCK_DB_DATA/library-items/date-and-time-selection/_date-and-time-selection.data.ts

```diff
@@ -0,0 +1,9 @@
+import { LibraryItem } from 'src/app/views/shared/models/library-item.model';
+
+import { YearSelection } from './year-selection';
+import { WeekSelection } from './week-selection';
+
+export const DateAndTimeSelectionData: LibraryItem[] = [
```

Comment:
We need to take another look at naming and organisation of components...

Reply:
> We need to take another look at naming and organisation of components...

Any specifics on this?